src/components/views/elements/Dropdown.js

```diff
@@ -329,4 +337,6 @@ Dropdown.propTypes = {
     // in the dropped-down menu.
     getShortOption: React.PropTypes.func,
     value: React.PropTypes.string,
+    // negative for consistency with HTML
+    disabled: React.PropTypes.bool,
```

Comment:
Does the appearance of the button look disabled when it's disabled?

Reply:
no, right now it looks the same